docs/en/observability/logs-stream.asciidoc

```diff
@@ -133,20 +133,299 @@ image::images/stream-logs-example.png[example search query on the logs stream pa
+
+This command creates an ingest pipeline with a dissect and a date processor:
+
+[source,console]
+----
+PUT _ingest/pipeline/logs-example-default
+{
+  "description": "Extracts the timestamp from log",
+  "processors": [
+    {
+      "dissect": {
+        "field": "message",
+        "pattern": "%{@timestamp} %{message}"
+      }
+    },
+    {
+      "date": {
+        "field": "@timestamp",
+        "formats": ["ISO8601"]
+      }
+    }
+  ]
+}
+----
+
+. Set these values for the dissect processor:
+- `field` – The field you're extracting data from, `message` in this case.
+- `pattern`– The pattern of the elements in your log data:
++
+[source,JSON]
+----
+%{timestamp} %{message}
+----
++
+This extracts the timestamp, `2023-08-08T13:45:12.123Z`, to the `timestamp` field, and the rest of the message, `WARN 192.168.1.101 Disk usage exceeds 90%.`, stays in the `message` field.
++
+. Set these values for the date processor:
+- `_ingest/pipeline/logs-example-default` – The name of the pipeline,`logs-example-default`, needs to match the name of your data stream. You'll set up your data stream in the next section. See {fleet-ref}/data-streams.html#data-streams-naming-scheme[data stream naming scheme] for more information.
+- `field` – The field you're extracting the date from, `@timestamp` in this case.
+- `formats` – The format of the current timestamp, `ISO8601` in this case. The date processor accepts {ref}/mapping-date-format.html[Java time patterns] or the following formats: `ISO8601`, `UNIX`, `UNIX_MS`, `TAI64N`.
+
+You can set additional fields with the date processor to update timezones, set a different output field, and change the output format of your timestamp. For more on this, see the {ref}/date-processor.html[date processor] documentation.
+
+[discrete]
+[[logs-stream-simulate-api]]
+==== Test your pipeline with the simulate pipeline API
+
+Make sure your pipeline is working as expected with the {ref}/simulate-pipeline-api.html#ingest-verbose-param[simulate pipeline API]. Run this command to test your pipeline:
+
+[source,console]
+----
+POST _ingest/pipeline/logs-example-default/_simulate
+{
+  "docs": [
+    {
+      "_source": {
+        "message": "2023-08-08T13:45:12.123Z WARN 192.168.1.101 Disk usage exceeds 90%."
+      }
+    }
+  ]
+}
+----
+
+The results should show the `@timestamp` field extracted from the `message` field:
+
+[source,console]
+----
+{
+  "docs": [
+    {
+      "doc": {
+        "_index": "_index",
+        "_id": "_id",
+        "_version": "-3",
+        "_source": {
+          "message": "WARN 192.168.1.101 Disk usage exceeds 90%.",
+          "@timestamp": "2023-08-08T13:45:12.123Z"
+        },
+        "_ingest": {
+          "timestamp": "2023-08-10T20:37:29.090624429Z"
```

Comment:
Was this timestamp automatically created based on the above pipeline?

Reply:
Yeah, this was the result I got when running the simulate pipeline api.

Reply:
Interesting. Lets leave it as this. Will dig into this on my end.